ios/FluentUI/Extensions/EnvironmentValues+Extensions.swift

```diff
@@ -7,7 +7,8 @@ import SwiftUI
 
 struct SafeAreaInsetsKey: EnvironmentKey {
     static var defaultValue: EdgeInsets {
-        UIApplication.shared.keyWindow?.safeAreaInsets.swiftUIInsets ?? EdgeInsets()
+        @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
```

Comment:
Wouldn't be simpler to just add a computed property directly in the NotificationViewSwiftUI View since it's got the windowProvider environment value?